ci-operator/config/openshift/oadp-operator/openshift-oadp-operator-oadp-1.3__4.11.yaml

```
@@ -0,0 +1,198 @@
+base_images:
+  rhel-8-golang-1.20:
+    name: builder
+    namespace: ocp
+    tag: rhel-8-golang-1.20-openshift-4.14
+build_root:
+  image_stream_tag:
+    name: builder
+    namespace: ocp
+    tag: rhel-8-golang-1.20-openshift-4.14
+images:
+- dockerfile_path: Dockerfile
+  from: rhel-8-golang-1.20
+  to: oadp-operator
+- dockerfile_path: build/ci-Dockerfile
+  from: rhel-8-golang-1.20
+  to: test-oadp-operator
+operator:
+  bundles:
+  - dockerfile_path: build/Dockerfile.bundle
+  substitutions:
+  - pullspec: quay.io/konveyor/oadp-operator:latest
+    with: oadp-operator
+releases:
+  latest:
+    release:
+      channel: fast
+      version: "4.11"
+resources:
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 1000m
+      memory: 512Mi
+tests:
+- as: operator-unit-test
+  steps:
+    test:
+    - as: unit
+      commands: HOME=$(pwd) make test
+      from: src
+      resources:
+        requests:
+          cpu: 100m
+          memory: 200Mi
+- as: operator-e2e-aws
+  steps:
+    cluster_profile: aws
+    dependencies:
+      OO_INDEX: ci-index
+    env:
+      OO_CHANNEL: stable-1.3
+      OO_INSTALL_NAMESPACE: openshift-adp
+      OO_PACKAGE: oadp-operator
+      OO_TARGET_NAMESPACES: '!install'
+    test:
+    - as: e2e
+      cli: latest
+      commands: CLUSTER_PROFILE=aws make test-e2e
+      credentials:
+      - mount_path: /var/run/oadp-credentials
+        name: oadp-credentials
+        namespace: test-credentials
+      from: test-oadp-operator
+      resources:
+        requests:
+          cpu: 1000m
+          memory: 512Mi
+    workflow: optional-operators-ci-aws
+- as: operator-e2e-azure
+  steps:
+    cluster_profile: azure4
+    dependencies:
+      OO_INDEX: ci-index
+    env:
+      OO_CHANNEL: stable-1.3
+      OO_INSTALL_NAMESPACE: openshift-adp
+      OO_PACKAGE: oadp-operator
+      OO_TARGET_NAMESPACES: '!install'
+    test:
+    - as: e2e
+      cli: latest
+      commands: CLUSTER_PROFILE=azure make test-e2e
+      credentials:
+      - mount_path: /var/run/oadp-credentials
+        name: oadp-credentials
+        namespace: test-credentials
+      from: test-oadp-operator
+      resources:
+        requests:
+          cpu: 1000m
+          memory: 512Mi
+    workflow: optional-operators-ci-azure
+- as: operator-e2e-gcp
+  steps:
+    cluster_profile: gcp
+    dependencies:
+      OO_INDEX: ci-index
+    env:
+      OO_CHANNEL: stable-1.3
+      OO_INSTALL_NAMESPACE: openshift-adp
+      OO_PACKAGE: oadp-operator
+      OO_TARGET_NAMESPACES: '!install'
+    test:
+    - as: e2e
+      cli: latest
+      commands: CLUSTER_PROFILE=gcp make test-e2e
+      credentials:
+      - mount_path: /var/run/oadp-credentials
+        name: oadp-credentials
+        namespace: test-credentials
+      from: test-oadp-operator
+      resources:
+        requests:
+          cpu: 1000m
+          memory: 512Mi
+    workflow: optional-operators-ci-gcp
+- as: operator-e2e-aws-periodic-slack
```

nit to be like other files
```suggestion
- as: operator-e2e-aws-periodic
```